src/client.rs

```diff
@@ -1141,9 +1166,14 @@ where
 
                     // Sync
                     // Frontend (client) is asking for the query result now.
```

Comment:
It would be nice to update these docs to include Flush. Something like:
```
                    // Sync (S)
                    // Frontend (client) is asking for the query result now.
                    // ... or ...
                    // Flush (H)
                    // Frontend (client) is asking for server to send query results now without
                    // sync.
```